src/django/api/models.py

```diff
@@ -117,6 +117,12 @@ class Contributor(models.Model):
         OTHER_CONTRIB_TYPE: 'Others',
     }
 
+    EMBED_LEVEL_CHOICES = (
```

Comment:
Consider adding a comment here explaining how these level choices relate to the named pricing tiers.

1 = Embed
2 = Embed+
3 = Both Embed Deluxe and Custom Embed (they have the same feature set but different facility quantity limits)

Reply:
Does it make sense to incorporate those descriptions into the tuple so that are used as labels in the admin, or are the raw numbers preferable there?

Reply:
I went ahead and implemented this.

Reply:
![image](https://user-images.githubusercontent.com/1042475/119542506-a010dd00-bd5d-11eb-92c0-b089dd821954.png)